### PR TITLE
Configurable gasLimit and gasEstimate bumping

### PR DESCRIPTION
**Pull request by pgebheim**

### CODE REVIEW

File: packages/ethersjs-provider/src/EthersProvider.ts [hunk 1]
```diff
@@ -38,6 +41,8 @@ export class EthersProvider extends ethers.providers.BaseProvider implements EPr
     }
   }
 
+  private
```
Comment (justinbarry) [inline packages/ethersjs-provider/src/EthersProvider.ts]: ?

Comment (pgebheim) [reply packages/ethersjs-provider/src/EthersProvider.ts]:
```suggestion
```

**Comment by pgebheim** [comment]
@justinbarry yah can just kill that private

**Comment by pgebheim** [comment]
@petong feel free to un WIP this if it works else I'll fix it more later

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). ✌️

 This link will always point to this PR's latest changes: https://peek.run/36whnn1

**Comment by pgebheim** [comment]
Removing WIP

**Review by petong — APPROVED** [review]

**Pull request merged**